README.md

```diff
@@ -45,8 +45,8 @@ The following workflow is for a new **Helm** operator:
 - [mercurial][mercurial_tool] version 3.9+
 - [docker][docker_tool] version 17.03+.
   - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
-- [kubectl][kubectl_tool] version v1.11.3+.
-- Access to a Kubernetes v1.11.3+ cluster.
+- [kubectl][kubectl_tool] version v1.12.0+.
+- Access to a Kubernetes v1.12.0+ cluster.
 - Optional: [delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) version 1.2.0+ (for `up local --enable-delve`).
 
```

Comment:
@estroz changed for 0.12 as we spoke about and tracked: https://github.com/operator-framework/operator-sdk/issues/2183